views/address.tmpl

```diff
@@ -12,12 +12,24 @@
 
                 <h4>Address</h4>
                 <p class="mono">{{.Address}}</p>
-
+                <p>
+                    {{if eq .NoOfTransactions .AddressRow}}
+                        Currently only the first {{.AddressRow}} transactions are being shown below.
+                    {{else}}
+                        Transactions: {{.NoOfTransactions}}</br>
+                        Unconfirmed: {{.TotalUnconfirmed}}</br>
+                        Received: {{.Received}}</br>
+                        Sent: {{.TotalSent}}</br>
+                        Unspent: {{.UnSpent}}</br>
+                    {{end}}
+                </p>
                 <h5>Transactions</h5>
                 <table class="table table-mono-cells table-sm striped">
                     <thead>
                         <th>Transactions ID</th>
-                        <th class="text-right">Total DCR</th>
+                        <th class="text-right">Received DCR</th>
+                        <th>Sent DCR</th>
+                        <th>Total DCR</th>
```

Comment:
What is "Total DCR" ?

Reply:
Its the total amount sent in the transaction, not address specific.